Portfolio.html

```diff
@@ -69,12 +71,13 @@ <h4>January 2021, Bristol UK</h4>
 
 
 	<div class="aboutme" id="aboutme">
-		<div class="container">
-			<div class="medbox">
-				<h2>About Me</h2>
-				<img src="jennyphoto.jpg" alt="Jenny Allton-Twist looking happy"/>
+		<div class="container aboutmebackground">
+			
+			<div class="medbox">	
```

Comment:
Perhaps consider a more descriptive class name eg mediumbox

Reply:
Thank you